freqtrade/commands/data_commands.py

```diff
@@ -64,8 +64,6 @@ def start_download_data(args: Dict[str, Any]) -> None:
     try:
 
         if config.get('download_trades'):
-            if config.get('trading_mode') == 'futures':
-                raise OperationalException("Trade download not supported for futures.")
```

Comment:
Please keep this here.

the ONLY reason to use freqtrade with download_trades (`--dl-trades` -> this mode) is on kraken - which is a far way from supporting futures (ccxt does not support kraken futures at the moment).